routes/favicon.coffee

```diff
@@ -0,0 +1,28 @@
+request = require 'request'
+fs = require 'fs'
+cachefile = "favicon.ico"
```

Comment:
Please match the style of the other files. In particular here, single quotes for strings which are not one of text for humans or strings with `#{}` interpolation. (Admittedly this style probably wasn't obvious.)